### ISSUE TITLE
Better block esp

### ISSUE THREAD

**Issue by birdman5923** [MeteorDevelopment/meteor-client#4769]
### Describe the feature

a module where I can specify block states in block esp
Specifically I want to differentiate between regular vaults and ominous vaults in 1.21, even though they are both the vault block

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client. (I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested. (I have searched suggestions on GitHub and read the FAQ)

**Comment by amongtrust22** [comment]
This isn't possible

**Comment by birdman5923** [comment]
Why not?

**Comment by vanvac** [comment]
its should be possible but probbably annoying to get it to work as they use the same block name as how they are stored the only thing that is different is the nbt data specifying if its state for example burnt out redstone torches when u esp them they both show but also for trial chambers there's like 10 rooms that can generate just memorise where the vaults are its not that hard(cough coming from someone that has memorised every ruined portal)
![image](https://github.com/user-attachments/assets/15efd10e-2611-461f-9b02-3a71eb3402da)
ominous = false

**Comment by vanvac** [comment]
![Screenshot 2024-08-02 103144](https://github.com/user-attachments/assets/bb826be6-9afe-444d-8c4b-4beefd9ac574)
![Screenshot 2024-08-02 103201](https://github.com/user-attachments/assets/e5a8217d-aebf-4c98-87fb-c1530f19783d)

**Comment by D-a-r-n-o-l-d** [comment]
yeah this would be super useful but i agree, prob not worth the time.

**Comment by Gingerbeard5773** [comment]
painful to code for mediocre feature
